index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Online Resume|About Me</title>
+	<link rel="stylesheet" type="text/css" href="styles.css/styles.css">
+	<a href=""></a>
+</head>
+<body>
+	<nav>
+<a><i>About Me</i></a> | <a href="#">Portfolio</a>
+	<h1>FAT JOE</h1>
+	<img src="https://upload.wikimedia.org/wikipedia/commons/1/1a/FatJoeApr2011.jpg" height="150px" width="150px">
+	</nav>
+
+<hr>
+
+<h3>I ENJOY LIFE AS A DEVELOPER</h3>
```

Comment:
please watch out for indentation mistakes.
Its difficult read structure of HTML